app/build.gradle

```
@@ -171,6 +197,9 @@ dependencies {
 
     implementation "androidx.palette:palette:$support_libraries_version"
 
+    jnaForTest "net.java.dev.jna:jna:${jna_version}@jar"
+    testImplementation files(configurations.jnaForTest.copyRecursive().files)
```

I just noticed we're missing the test-only include:
```suggestion
    testImplementation files(configurations.jnaForTest.copyRecursive().files)
    
    testImplementation "org.mozilla.telemetry:glean-forUnitTests:${project.ext.glean_version}"
```

---

(thanks @badboy )

---

Thanks! I couldn't figure out where `glean_version` was coming from since it was never declared in Fenix as well.

I learnt through some trial and error that it's from within the gradle plugin, so applying the plugin therefore needs to be done before the `dependencies` block. :)

---

Good point, I'll extended our docs to cover that too